### PR TITLE
Update intellifire.markdown

### PR DESCRIPTION
**Pull request by jeeftor**
Adding error states for PR

## Proposed change

Description of all the new binary error sensors

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/68249
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by mdegat01** [comment]
@jeeftor you merged `current` into your branch, hence the flood of commits. You need to rebase your branch on `current` to switch branches.

**Comment by jeeftor** [comment]
ooooops!

**Comment by jeeftor** [comment]
Are we good to merge this one? (the PR it is assigned to has the **SMASH** label on it?)

**Comment by thecode** [comment]
This PR is marked as `Added documentation for a new feature I'm adding to Home Assistant (next branch).` and is also linked to a new feature on `core` repo, so it should be targeted to `next` branch.
PR is not approved yet, so can't be merged, after you target it to the correct branch it will be reviewed.
Thanks

comment by @mdegat01 seems incorrect, unless I am missing something.

**Comment by jeeftor** [comment]
Gotcha - the bot threw me off when it said: 
image

it initially was against the next branch... I'll fix it.

File: source/_integrations/intellifire.markdown [hunk 1]
```diff
@@ -46,3 +46,20 @@ The following sensors are available as either a **Binary Sensor** when dealing w
...
+ - `ECM_OFFLINE` - ECM is offline.
+ - `FAN` - Fan Error. Your appliance has detected that an accessory is not functional. Please contact your dealer and report this issue.
+ - `FAN_DELAY` - Fan Information: Fan will turn on within 3 minutes. Your appliance has a built-in delay that prevents the fan from operating within the first 3 minutes of turning on the appliance. This allows the air to be heated prior to circulation.
+ - `FLAME` - Pilot Flame Error. Your appliance has been safely disabled. Please contact your dealer and report this issue.
+ - `LIGHTS` - Lights Error. Your appliance has detected that an accessory is not functional. Please contact your dealer and report this issue.
+ - `MAINTENANCE` - Maintenance: Your appliance is due for a routine maintenance check. Please contact your dealer to ensure your appliance is operating at peak performance.
+ - `OFFLINE` - Your appliance is currently offline.
+ - `PILOT_FLAME` - Pilot Flame Error: Your appliance has been safely disabled. Please contact your dealer and report this issue.
+ - `SOFT_LOCK_OUT` - Sorry your appliance did not start. Try again by pressing Flame ON.
```
Comment (frenck) [inline source/_integrations/intellifire.markdown]: Left a late review on the original PR, here:

<https://github.com/home-assistant/core/pull/67403#pullrequestreview-911339890>

These should not be upper case, and translations are missing.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/intellifire.markdown (hunk 1 above)
Comment (jeeftor) [reply source/_integrations/intellifire.markdown]: Do I still need translations as they are now binary sensors? I'm thinking probably not

Comment (jeeftor) [reply source/_integrations/intellifire.markdown]: https://github.com/home-assistant/core/pull/68249

Here is the new PR

**Review by frenck — APPROVED** [review]
Thanks, @jeeftor 👍

**Pull request merged**